<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeEditDialog</class>
 <widget class="QDialog" name="RecipeEditDialog">
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рецепт производства</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название рецепта:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Название рецепта</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputGoodLabel">
       <property name="text">
        <string>Продукция на выходе:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="outputGoodCombo">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="outputQtyLabel">
       <property name="text">
        <string>Выход (кол-во на порцию):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="outputQtySpin">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001</double>
       </property>
       <property name="maximum">
        <double>999999.999</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="componentsHeaderLabel">
     <property name="text">
      <string>Состав рецепта (норма на одну порцию выхода):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="componentsTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="columnCount">
      <number>2</number>
     </attribute>
     <column>
      <property name="text">
       <string>Сырьё (товар)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество на порцию</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="compLayout">
     <item>
      <widget class="QLabel" name="compGoodLabel">
       <property name="text">
        <string>Сырьё:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="componentGoodCombo">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="compQtyLabel">
       <property name="text">
        <string>Кол-во:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="componentQtySpin">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001</double>
       </property>
       <property name="maximum">
        <double>999999.999</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addCompBtn">
       <property name="text">
        <string>Добавить сырьё</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeCompBtn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
